src/masternodes/loan.h

```diff
@@ -135,7 +136,7 @@ struct CVaultMessage {
 
 struct CVault : public CVaultMessage
 {
-    bool isLiquidated{false};
+    CVaultId id{};
 
     ADD_SERIALIZE_METHODS;
 
```

Comment:
It's a pointless struct now. Vault id should not be serialized nor part of value, it's the key otherwise it's waste of storage.